source/TradingPanel.cpp

```diff
@@ -337,3 +336,79 @@ void TradingPanel::Buy(int64_t amount)
 	player.Accounts().AddCredits(-amount * price);
 	GameData::AddPurchase(system, type, amount);
 }
+
+
+
+void TradingPanel::SellOutfitsOrMinables(bool sellMinable)
+{
+	int day = player.GetDate().DaysSinceEpoch();
+	for(const auto &it : player.Cargo().Outfits())
+	{
+		if(sellMinable != static_cast<bool>(it.first->Get("minable")))
+			continue;
```

Comment:
```suggestion
			continue;
		if(!it.second)
			continue;
```
either add it here too or remove from the other place below

Reply:
That is special handling for when the `Outfit*` in the Outfits map is nullptr. I'm not sure that's possible.

Reply:
idk, I think I saw it once in the existing code

Reply:
The advantage of selling the nullptrs is that it cleans them out of the map.

The disadvantage is that I don't understand why they'd be in there in the first place, so I don't know how to correctly handle them, if they exist at all.

Reply:
It's not for nullptrs, it's when you have 0 outifts. Looks like our current code isn't consistent on this matter, but CargoHold methods used for transferring outfits don't clear empty entries, so this check saves some unnecessary calls to depreciation and other stuff.